python/lsst/pipe/analysis/plotRhoStatistics.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+from matplotlib import pyplot as plt
+import treecorr
+
+from lsst.daf.persistence import Butler
+import lsst.afw.table as afwTable
+from utils import E1, E2, E1Resids, E2Resids, E1ResidsHsmRegauss, E2ResidsHsmRegauss, PsfTraceSizeDiff
+
+def corr2(ra, dec, g1a, g2a, g1b, g2b, ra_units="degrees", dec_units="degrees", **treecorr_kwargs):
```

Comment:
While we are now allowed to use snake_case it is probably good to be consistent about it. Other variables are in camelCase, it would be nicer if ra_units etc were consistent with other variables.

Reply:
I was trying to avoid the confusion between treecorr referring it to as ra_units and the pipe analysis code referring it to as raUnits. I am not strongly opinionated here.